locales/en/messages.json

```diff
@@ -2551,6 +2551,9 @@
     "motorsEnableControl": {
         "message": "<strong>I understand the risks</strong>, the propellers are removed - enable motor control and arming, and disable Runaway Takeoff Prevention."
     },
+    "motorsDialogMixerReset": {
+        "message": "<strong>Invalid mixer model selected.</strong><br /><br />The selected mixer model needs {{mixerMotors}} motor resources.<br /><br />With the current configuration you are not able to test any motors or fly your model.<br /><br />Resetting to default mixer model"
```

Comment:
It's always good add a description field to any translation, to let know the translators where is this being used. Is not mandatory, but usually it helps.